embuilder.py

```diff
@@ -66,6 +66,7 @@
     'sdl2-mt': ('sdl2', {'USE_PTHREADS': 1}),
     'icu-mt': ('icu', {'USE_PTHREADS': 1}),
     'sdl2_mixer_mp3': ('sdl2_mixer', {'SDL2_MIXER_FORMATS': ["mp3"]}),
+    'sdl2_mixer_mid_mp3_ogg': ('sdl2_mixer', {'SDL2_MIXER_FORMATS': ["mid", "mp3", "ogg"]}),
     'sdl2_mixer_none': ('sdl2_mixer', {'SDL2_MIXER_FORMATS': []}),
```

Comment:
drive by addition...

Reply:
It looks like we build all the port variants on CI on linux. Each mixer variant takes 2-3 seconds or so. So this PR adds a little, but probably not worth worrying about considering the other stuff we build... but it would be good eventually to slim things down.

Reply:
Thinking about how to allow embuilder CLI to build any variant without manually listing them here (and then requiring CI to build every combo...)

options i see are

1) do what the comment above suggests, and have each port declare what ports they use (can have a common function that takes each variant tag and produces a combination for each). to prevent CI from doing _everything_ could also maintain a list of variants to actually run for CI
2) allow embuilder to accept a settings flag, instead of hardcoding them here

I think the second option sounds promising...what do you think?

Reply:
We maintain a list of things are not build automatically.  See `skip_tasks = ['cocos2d']` below.  I would just add these to that list.

The commend for that line is not accurate, but I happen to be updating it right now: https://github.com/emscripten-core/emscripten/pull/16747/files